### PR TITLE
SCT-1770 continue removing Jongo layer from type database

### PR DESCRIPTION
**Pull request by MrCreosote**
Added tests for uncovered code prior to changes

### CODE REVIEW

File: src/us/kbase/typedobj/db/MongoTypeStorage.java [hunk 1]
```diff
@@ -54,36 +54,45 @@ public MongoTypeStorage(DB db) {
...
-		MongoCollection frefs = jdb.getCollection(TABLE_FUNC_REFS);
-		frefs.ensureIndex("{depModule:1,depName:1,depVersion:1}", "{unique:false}");
-		frefs.ensureIndex("{refModule:1,refName:1,refVersion:1}", "{unique:false}");
-		frefs.ensureIndex("{depModule:1,depModuleVersion:1}", "{unique:false}");
-		MongoCollection trefs = jdb.getCollection(TABLE_TYPE_REFS);
-		trefs.ensureIndex("{depModule:1,depName:1,depVersion:1}", "{unique:false}");
-		trefs.ensureIndex("{refModule:1,refName:1,refVersion:1}", "{unique:false}");
-		trefs.ensureIndex("{depModule:1,depModuleVersion:1}", "{unique:false}");
+		final DBCollection reqs = db.getCollection(TABLE_MODULE_REQUEST);
+		reqs.createIndex(new BasicDBObject("moduleName", 1).append("ownerUserId", 1), UNIQ);
```
Comment (scanon) [inline src/us/kbase/typedobj/db/MongoTypeStorage.java]: Do these raise any error if the index already exists?

File: src/us/kbase/workspace/test/database/mongo/MongoWorkspaceDBTest.java [hunk 2]
```diff
@@ -76,6 +76,7 @@ public static void tearDownClass() throws Exception {
 			MONGO.destroy(TestCommon.getDeleteTempFiles());
 		}
 	}
+
```
Comment (scanon) [inline src/us/kbase/workspace/test/database/mongo/MongoWorkspaceDBTest.java]: Must better.

File: src/us/kbase/typedobj/db/MongoTypeStorage.java (hunk 1 above)
Comment (MrCreosote) [reply src/us/kbase/typedobj/db/MongoTypeStorage.java]: Nope

Comment (MrCreosote) [reply src/us/kbase/typedobj/db/MongoTypeStorage.java]: Oh, I see why you were thinking that. They deprecated `ensureIndex` and made `createIndex` the replacement.

**Pull request merged**